component/rules.jsonnet

```
@@ -2,6 +2,10 @@ local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
 local params = inv.parameters.rancher_monitoring;
 local ignoreNames = params.alerts.ignoreNames;
+local customAnnotations = params.alerts.customAnnotations;
+local defaultAnnotations = {
+  syn_component: 'rancher-monitoring',
```

```suggestion
  syn_component: inv.parameters._instance,
```

the `_instance` parameter is set to the component name (if not multi instantiated) see https://github.com/projectsyn/documentation/blob/master/docs/modules/SDDs/pages/0025-commodore-component-instantiation.adoc for further details